shared/src/commonMain/kotlin/com/intive/picover/parties/view/AddPartyContent.kt

```diff
@@ -100,27 +98,3 @@ fun AddPartyContent(
 		}
 	}
 }
-
-@Preview(showBackground = true)
-@Composable
-private fun AddPartyContentValidPreview() {
```

Comment:
Previews will be declared in platform specific source sets? Or what's the plan here?